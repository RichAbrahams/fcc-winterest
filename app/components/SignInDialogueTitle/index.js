/**
*
* SignInDialogueTitle
*
*/

import React from 'react';
import Title from './styles';


function SignInDialogueTitle(props) {
  return (
    <Title>
      {props.children}
    </Title>
  );
}

SignInDialogueTitle.propTypes = {
  children: React.PropTypes.string,
};

export default SignInDialogueTitle;
