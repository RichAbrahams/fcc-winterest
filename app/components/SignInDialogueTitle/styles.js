import styled from 'styled-components';

const Title = styled.h2 `
  font-family: 'Joti One', cursive;
  color: white;
  text-align: center;
  margin: 0rem 0rem 1rem 0rem;
`;

export default Title;
