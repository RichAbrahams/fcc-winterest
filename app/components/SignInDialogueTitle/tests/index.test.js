
import React from 'react';
import renderer from 'react-test-renderer';
import SignInDialogueTitle from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <SignInDialogueTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});

