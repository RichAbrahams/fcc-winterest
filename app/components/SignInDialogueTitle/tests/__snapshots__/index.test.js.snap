exports[`test renders correctly 1`] = `
<h2
  className="cTfNJF" />
`;
